SymmetricProject/finsums.lean

```diff
@@ -1,21 +1,16 @@
 -- This is not part of the project, but was written in order for to practice manipulation of finite sums in Lean.
 
-import Mathlib.Data.Real.Basic
 import Mathlib.Data.Finset.Basic
-import Mathlib.RingTheory.MvPolynomial.Basic
-import Mathlib.RingTheory.MvPolynomial.Symmetric
 import Mathlib.Algebra.BigOperators.Basic
+import Mathlib.Tactic.Linarith
 
-open MvPolynomial
 open Finset
 open BigOperators
 
 example (n : ℕ) : ∑ t in range n, 2*t + n = n * n := by
   induction' n with n hn
-  simp
-  simp [Finset.sum_range_succ, hn]
-  rw [Nat.succ_eq_add_one]
-  have h : (∑ t in range n, 2 * t) + 2 * n + (n + 1) = (∑ t in range n, 2 * t) + n + n + (n + 1) := by ring 
-  rw [h, hn]
-  ring
-  
+  · simp
+  · simp [Finset.sum_range_succ]
+    linarith
```

Comment:
Since the argument is not actually using much about multiplication here, I guessed that `linarith` would be able to cope here. Even better, because `linarith` can use local hypotheses, there is no need to separately state `h` or rewrite by `hn`.